pkgs/development/python-modules/gower/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  setuptools,
+  fetchPypi,
+  scipy,
+  pandas,
+}:
+
+buildPythonPackage rec {
+
+  pname = "gower";
+  version = "0.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-NN21FY8Oi/ugk9yga5+Ie9okSZjRCvKjrYx0pu+htfY=";
+  };
+
+  patchPhase = ''
```

Comment:
Please do not overwrite `patchPhase` because it will prevent us from applying patches.

Reply:
I have PR'ed the patch you suggested, by supposing we had to something like this, which phase should I have used instead? postPatch?

Reply:
Yes, we prefer to use `postPatch` instead.